InscryptionAPI/Card/TribeManager.cs

```diff
@@ -0,0 +1,112 @@
+using DiskCardGame;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using UnityEngine;
+using HarmonyLib;
+using InscryptionAPI.Guid;
+using InscryptionAPI.Helpers;
+
+namespace InscryptionAPI.Card
+{
+    [HarmonyPatch]
+    public class TribeManager
+    {
+        private static readonly List<TribeInfo> tribes = new();
+
+        [HarmonyPatch(typeof(CardDisplayer3D), "UpdateTribeIcon")]
+        [HarmonyPostfix]
+        public static void UpdateTribeIcon(CardDisplayer3D __instance, CardInfo info)
+        {
+            foreach (TribeInfo tribe in tribes)
+            {
+                if (tribe.icon != null)
+                {
+                    if (info.IsOfTribe(tribe.tribe))
+                    {
+                        bool foundSpriteRenderer = false;
+                        foreach (SpriteRenderer spriteRenderer in __instance.tribeIconRenderers)
+                        {
+                            if (spriteRenderer.sprite == null)
+                            {
+                                foundSpriteRenderer = true;
+                                spriteRenderer.sprite = tribe.icon;
+                                break;
+                            }
+                        }
+                        if (!foundSpriteRenderer)
+                        {
+                            SpriteRenderer last = __instance.tribeIconRenderers.Last();
+                            SpriteRenderer spriteRenderer = UnityEngine.Object.Instantiate(last);
+                            spriteRenderer.transform.parent = last.transform.parent;
+                            spriteRenderer.transform.localPosition = last.transform.localPosition + (__instance.tribeIconRenderers[1].transform.localPosition - __instance.tribeIconRenderers[0].transform.localPosition);
+                        }
+                    }
+                }
+            }
+        }
+
+        [HarmonyPatch(typeof(CardSingleChoicesSequencer), "GetCardbackTexture")]
+        [HarmonyPostfix]
+        public static void GetCardbackTexture(ref Texture __result, CardChoice choice)
+        {
+            if (__result == null && choice.resourceType != ResourceType.Blood && choice.resourceType != ResourceType.Bone)
+            {
+                __result = tribes.Find((x) => x.tribe == choice.tribe).cardback;
+            }
+        }
+
+        [HarmonyPatch(typeof(Part1CardChoiceGenerator), "GenerateTribeChoices")]
+        [HarmonyPrefix]
+        public static bool GenerateTribeChoices(ref List<CardChoice> __result, int randomSeed)
+        {
+            List<Tribe> list = new()
```

Comment:
Consider `new(Enum.GetValues<Tribe>)`

Reply:
i took the code from base game and also all values has the none and NUM_TRIBES values